addons/website_crm_partner_assign/views/website_crm_partner_assign_templates.xml

```diff
@@ -54,52 +52,118 @@
                     </li>
                 </t>
             </ul>
+        </aside>
 
-        </div>
+        <!-- Off canvas filters on mobile-->
+        <aside class="filters_offcanvas offcanvas offcanvas-end d-lg-none p-0 overflow-visible">
+            <div class="accordion accordion-flush">
+                <div class="accordion-item">
+                    <h2 class="accordion-header">
+                        <button class="accordion-button border-top fw-bold collapsed"
+                            type="button"
+                            data-bs-toggle="collapse"
+                            data-bs-target=".offcanvas_grade"
+                            aria-expanded="false"
+                            aria-controls="offcanvas_grade">
+                            Filter by Level
+                        </button>
+                    </h2>
+                    <div class="offcanvas_grade accordion-collapse collapse">
+                        <div class="accordion-body">
+                            <ul id="reseller_grades" class="nav nav-pills flex-column">
+                                <t t-foreach="grades" t-as="grade">
+                                    <li class="nav-item">
+                                        <a t-attf-href="/partners#{ grade['grade_id'][0] and '/grade/%s' % grade['grade_id'][0] or '' }#{ current_country and '/country/%s' % slug(current_country) or '' }#{ '?' + (search_path or '') + '&amp;' + keep_query('country_all') }"
+                                        t-attf-class="nav-link#{grade['active'] and ' active' or ''}">
+                                            <span class="badge rounded-pill float-end" t-esc="grade['grade_id_count'] or ''" />
+                                            <t t-esc="grade['grade_id'][1]"/>
+                                        </a>
+                                    </li>
+                                </t>
+                            </ul>
+                        </div>
+                    </div>
+                </div>
+                <div class="accordion-item">
+                    <h2 class="accordion-header">
+                        <button class="accordion-button border-top fw-bold collapsed"
+                            type="button"
+                            data-bs-toggle="collapse"
+                            data-bs-target=".offcanvas_country"
+                            aria-expanded="false"
+                            aria-controls="offcanvas_country">
+                            Filter by Country
+                        </button>
+                    </h2>
+                    <div class="offcanvas_country accordion-collapse collapse">
+                        <div class="accordion-body">
+                            <ul id="reseller_countries" class="nav nav-pills flex-column">
```

Comment:
My eyes are bleeding... can we use `list-group list-group-flush` and `list-group-item list-group-item-action` instead? On desktop too maybe ?

ref
https://getbootstrap.com/docs/5.1/components/list-group/#flush
https://getbootstrap.com/docs/5.1/components/list-group/#links-and-buttons